docs/capabilities.md

```diff
@@ -166,6 +166,7 @@
 * `config => call => start-without-media` (local) - Boolean, whether media should be disabled when starting or joining a conversation
 * `config => call => max-duration` - Integer, maximum call duration in seconds. Please note that this should only be used with system cron and with a reasonable high value, due to the expended duration until the background job ran.
 * `config => call => blur-virtual-background` (local) - Boolean, whether blur background is set by default when joining a conversation
++ `edit-draft-poll` - Whether moderators can edit draft polls
```

Comment:
Move to 21? Or are we backport this? Not sure

Reply:
we can backport if you like? It's up to you.

Reply:
We never backport here (unless forced, and I see no one forcing here) :speak_no_evil: